app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreenViewModel.kt

```diff
@@ -9,9 +9,9 @@ class SplashScreenViewModel(
     private val users: Users,
 ) : ViewModel() {
 
-    fun migratePreferences() {
+    fun migratePreferencesIfNeeded() {
         preferencesMigrator.migrateIfNeeded()
     }
 
-    suspend fun requiresInitialSetup(): Boolean = users.getPowerUser() == null
+    suspend fun doesRequireInitialSetup(): Boolean = users.getPowerUser() == null
```

Comment:
I'm guessing you updated this to sound more like it returns a boolean, but I think it still sounds like a statement. Can reword this to start with "is"?

Reply:
what about 'isInitialSetupRequired' ?